src/borg/archiver.py

```diff
@@ -78,8 +78,10 @@ def wrapper(self, args, **kwargs):
             elif location.proto == 'ssh':
                 repository = RemoteRepository(location, create=create, lock_wait=self.lock_wait, lock=lock, args=args)
```

Comment:
The client currently has no notion of append_only, and I don't think we could implement it without the server's assistance. I'll just add a comment then.